pkg/query-service/dao/postgres/rbac.go

```diff
@@ -0,0 +1,520 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"go.signoz.io/query-service/model"
+	"go.signoz.io/query-service/telemetry"
+)
+
+func (mds *modelDao) CreateInviteEntry(ctx context.Context,
+	req *model.InvitationObject) *model.ApiError {
```

Comment:
What I initially thought is this `modelDao` or something else would be responsible for working with different dialects and these methods would be defined on `Invitation` or `Org` objects.